templates/Web/Pages/Angular.Grid/src/app/app-shell/AngularGrid/grid.component.ts

```diff
@@ -25,7 +26,11 @@ export class GridComponent implements OnInit {
       id: 1
     }
   ];
-  constructor(private gridService: GridService) { }
+  constructor(private gridService: GridService, iconRegistry: MatIconRegistry, sanitizer: DomSanitizer) {
+    iconRegistry.addSvgIcon(
+      'grey-box',
+      sanitizer.bypassSecurityTrustResourceUrl('assets/GreyBox.svg'));
```

Comment:
What does `bypassSecurityTrustResourceUrl` mean?

Reply:
It is a default method. Cause this module uses HTTP Request to get svg images. I guess here it tries to creates a URL for this svg image.